src/util/InputView.js

```diff
@@ -0,0 +1,62 @@
+const { Console } = require("@woowacourse/mission-utils");
+const { INPUT_MESSAGE } = require("./Constant");
+
+/**
+ * @param {object} error 발생한 에러 객체
+ * @param {function} recursiveFunction 호출할 함수
+ * @param {function} callback recursiveFunction 전달할 callback 함수
+ */
+const handleError = (error, recursiveFunction, callback) => {
+  Console.print(error.message);
+  recursiveFunction(callback);
+};
```

Comment:
```suggestion
const handleError = (answer, recursiveFunction, callback) => {
  try {
    callback(answer);
  } catch (error) {
    Console.print(error.message);
    recursiveFunction(callback);
};
```
이런 식으로  try-catch부분도 반복되니까 handleError에서 처리해주는 것이 어떤가요?

Reply:
handleError에서는 에러가 발생했을 때 일어날 일들만 정의하려고 따로 분리했습니다!